cpp/tests/binning/bin_tests.cpp

```diff
@@ -0,0 +1,429 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/binning/bin.hpp>
+#include <cudf/column/column.hpp>
+#include <cudf/column/column_view.hpp>
+#include <cudf/types.hpp>
+#include <cudf_test/base_fixture.hpp>
+#include <cudf_test/column_wrapper.hpp>
+#include <cudf_test/type_list_utilities.hpp>
+#include <cudf_test/type_lists.hpp>
+
+#include <algorithm>
+#include <limits>
+#include <numeric>
+#include <vector>
+
+namespace {
+
+using namespace cudf::test;
```

Comment:
Is this being used?

Reply:
Not much anymore, but yes. Before I cleaned up this file all the individual tests had their own wrapper declarations etc that required access to cudf::test. Now it's just that the other typedefs just below this become a bit more cumbersome without it.

Reply:
I'm just not seeing where that is the case in this file. All the types below seem fully-qualified correctly. I would recommend removing this.

Reply:
`Concat` and the various type lists (`FloatingPointTypes` and `FixedPointTypes` in particular are also used in specific tests below) are not consistently qualified. But I'm in favor of not using namespaces where it's not necessary, so I can fully qualify everything and remove this.